doc/api/cli.md

```diff
@@ -530,6 +530,22 @@ changes:
 Specify the `module` of a custom experimental [ECMAScript module loader][].
 `module` may be any string accepted as an [`import` specifier][].
 
+### `--experimental-noderealm`
```

Comment:
Maybe it’s just me, but this really feels like it should be spelled `--experimental-node-realm`.